CHANGELOG.md

```diff
@@ -18,6 +18,11 @@
 ### Fixed
 
 - Fixed a bug where a worker's `World` could get disposed multiple times if you stopped the application inside the Editor while the worker is being created.
+- Fixed a bug when uploading assemblies with the deployment launcher window, by adding the `--enable_pre_upload_check=false` argument.
```

Comment:
What bug was this exactly?

Reply:
a bug that only exists in our develop branch afaik, so I don't think a changelog entry is necessary

Reply:
will remove changelog entry - it's the issue where spatial `prepare for run` happens and breaks because our schema isn't all in one folder now